lib/model/model.go

```diff
@@ -1530,7 +1531,7 @@ func (m *Model) numHashers(folder string) int {
 // generateClusterConfig returns a ClusterConfigMessage that is correct for
 // the given peer device
 func (m *Model) generateClusterConfig(device protocol.DeviceID) protocol.ClusterConfigMessage {
-	cm := protocol.ClusterConfigMessage{
+	clusterConfigMessage := protocol.ClusterConfigMessage{
```

Comment:
I'll quibble once over the names here. I'm all on board with the existing names being too cryptic, but how about `message` (we know what type of message we are building, `folder` (we know it's for the protocol) and `device` (same)?